src/components/styles/_neon-border.less

```diff
@@ -13,6 +13,12 @@
     }
   }
 
+  [data-theme="dark"] &,
+  &.dark-mode,
+  &:hover:not(.white-btn) {
```

Comment:
can you also add an exception for `yellow-btn` just so that it will be consistent with other buttons.

Reply:
just to be clear on hover yellow btn should have black color in day mode? if not pls elaborate

Reply:
In both modes yellow-btn should have the color black on hover